tf/main.tf

```diff
@@ -0,0 +1,25 @@
+terraform {
+  required_version = "~>1.0"
+
+  backend "s3" {
```

Comment:
Doesn't the organisation of this file, not comply with the general rules? Isn't provider usually at the top etc

Reply:
This follows the same structure as what's in the current cvs-tf repo.